src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.db;
+
+import core.basesyntax.FruitTransaction;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Storage {
+
+    public static final List<FruitTransaction> storageFromInputFile = new ArrayList<>();
+
```

Comment:
you should have only 1 DB
![image](https://github.com/mate-academy/jv-fruit-shop/assets/73041091/3765af12-b687-4f2c-bd28-b4e2e47368b5)

```suggestion
```